### PR TITLE
Added: Recreated Chicken Stirfry

### PR DESCRIPTION
**Pull request by RohanVB**

### CODE REVIEW

File: cfc/1-RohanVB-recreated_chicken_stirfry.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+Alternatively, you could grill the vegetables separately and  
+mince them after they are cooked.  
+
+  
+
+Congratulations, you have prepared an awesome dish that serves 1  
+
+person, because it's not like you're gonna have anyone over  
+
+anyways. D:
```
Comment (adhikasp) [inline cfc/1-RohanVB-recreated_chicken_stirfry.md]: +1 for burn factor

**Comment by Makman2** [comment]
Hahaha actually I thought this repo was for code recipes, seems I've totally misunderstood things :D :D :D

**Comment by RohanVB** [comment]
sims blogged about it :D 
IMO it'd be way worse if it was a repo for code recipes and I made this as the first contributing PR :p

**Comment by meetmangukiya** [comment]
Too much whitespace, don't use newline where not required :P

**Comment by meetmangukiya** [comment]
Squash the commits!

**Comment by NiklasMM** [comment]
Included in #8 :blush:

**Pull request closed without merging**